Google.PowerShell/BigQuery/BqDataset.cs

```diff
@@ -272,8 +276,10 @@ protected override void ProcessRecord()
     ///   <para>This takes a Dataset object from the pipeline and inserts it into the Cloud project "my-project".</para>
     /// </example>
     /// <example>
-    ///   <code>PS C:\> New-BqDataset "test_data_id" -Name "Testdata" `
-    ///   -Description "Some interesting data!" -Expiration 86400000</code>
+    ///   <code>
+    /// PS C:\> New-BqDataset "test_data_id" -Name "Testdata" `
+    /// -Description "Some interesting data!" -Expiration 86400000
```

Comment:
This line should be indented, so it looks more like

    PS C:> New-BqDataset "test_data_id" ...
                         -Description ...

You may want to add one more line break if you do so.